install.sh

```diff
@@ -254,13 +254,29 @@ libsodium_install ()
 joinmarket_install ()
 {
     jm_pkgs=( 'jmbase' 'jmdaemon' 'jmbitcoin' 'jmclient' )
+    rm -rf "${jm_root}/lib/python2.7/site-packages/easy-install.pth"
```

Comment:
Just showing my ignorance on easy-install here, but what does this do?

Reply:
It's to force remove (actually could've used just `-f`) a file called `easy-install.pth` inside `jmvenv`.  It'll be there if you passed `--develop` to `install.sh`.  If at some point you want to re-run `install.sh` without `--develop`, you need this file removed otherwise `pip` still recognizes jm-cs as installed.  Basically that file contains the path to jm-cs's `.py` files.
The next `rm -rf` removes the joinmarket package itself that's in `jmvenv` and that can have either `info` or `link`.  I think I should also include these suffixes verbatim, so don't merge yet in any case.  I'll look into this tomorrow.

Reply:
> If at some point you want to re-run install.sh without --develop, you need this file removed otherwise pip still recognizes jm-cs as installed.

Ah! Thanks for that - I think this may have tripped me up in the past (although don't really remember).